design-proposals/emts-scale-provisioning.md

```diff
@@ -0,0 +1,27 @@
+# Design Proposal: Scale provisioning of EMT-S edge node supporting OXM workflow
+
+Author(s): EIM team.
+
+Last updated: 29.04.2025
+
+## Abstract
+
+The Edge Microvisor Toolkit Standalone (EMT-S) node is designed to enable enterprise customers and developers to evaluate Intel silicon-based platforms for Edge AI use cases. In this context, Original Equipment Manufacturers (OXMs) play a critical role by preparing edge nodes in bulk for end customers at their facilities before shipping them to the deployment locations. To support the OXM workflow for Edge AI use cases, it is essential to implement a scalable provisioning solution for multiple EMT-S nodes. This document outlines the design proposal for enabling large-scale provisioning of EMT-S edge nodes to meet the requirements of the OXM workflow.
+
+### Proposal Summary
```

Comment:
Should this proposal include what are the components of EIM-S that will be deployed on-prem? CC: @daniele-moro

Reply:
I believe we should not deploy infra-managers at all

Reply:
IMO, yes, we should clarify which components of EIM we really need to deploy. I guess API, Inventory, OM, TC (?), DKAM (?).
We may not care about multi-tenancy for this use case, but we are currently tightly coupled with TC+OSRM for tenant bootstrap, in order for EMF-EIM to work. But we could figure out a way to overcome this limitation.
Also, we are currently tightly coupled with multiple platform provided components (DB, Keycloak, Vault...).

Reply:
let's have this as a separate ADR, for the scope of the e2e provisioning it does not matter if we have 1 or 100 components and FPS elements.